apps/python-docs/components/markdown/callout.tsx

```diff
@@ -0,0 +1,88 @@
+import { Icon } from '@/components/markdown/icons/Icon';
+
+const styles = {
+    note: {
+        container: 'bg-sky-50 dark:bg-slate-800/60 dark:ring-1 dark:ring-slate-300/10',
+        title: 'text-sky-900 dark:text-sky-400',
+        body: 'text-sky-800 [--tw-prose-background:theme(colors.sky.50)] prose-a:text-sky-900 prose-code:text-sky-900 dark:text-slate-300 dark:prose-code:text-slate-300'
+    },
+    warning: {
+        container: 'bg-amber-50 dark:bg-slate-800/60 dark:ring-1 dark:ring-slate-300/10',
+        title: 'text-amber-900 dark:text-amber-500',
+        body: 'text-amber-800 [--tw-prose-underline:theme(colors.amber.400)] [--tw-prose-background:theme(colors.amber.50)] prose-a:text-amber-900 prose-code:text-amber-900 dark:text-slate-300 dark:[--tw-prose-underline:theme(colors.sky.700)] dark:prose-code:text-slate-300'
+    }
+};
+
+const icons = {
+    note: (props) => <Icon icon="lightbulb" {...props} />,
+    warning: (props) => <Icon icon="warning" color="amber" {...props} />
+};
+
+export function Callout({ type = 'note', title, children }) {
+    let IconComponent = icons[type];
+
+    return (
+        <div className={`my-8 flex rounded-3xl p-6 ${styles[type].container}`}>
```

Comment:
(nit) For interpolation of classnames we have a utility `cn` that uses `tailwind-merge` to properly handle merging tailwind classes.

For these cases it's not really that important, but it's particularly important when classes can be set arbitrarily.

Reply:
I've got a turborepo newbie question for you -- since it's in a different sub-site, how do I import it?